gpt4all-chat/download.cpp

```diff
@@ -109,7 +109,9 @@ void Download::downloadModel(const QString &modelFile)
             = QString("ERROR: Could not open temp file: %1 %2").arg(tempFile->fileName()).arg(modelFile);
         qWarning() << error;
         clearRetry(modelFile);
-        ModelList::globalInstance()->updateDataByFilename(modelFile, ModelList::DownloadErrorRole, error);
+        QVector<QPair<int, QVariant>> data;
+        data.append(qMakePair(ModelList::DownloadErrorRole, error));
+        ModelList::globalInstance()->updateDataByFilename(modelFile, data);
```

Comment:
Is there any reason we aren't using initializer lists, at least for the single-pair updates? And I don't think we need to explicitly call qMakePair anywhere.
```suggestion
        ModelList::globalInstance()->updateDataByFilename(modelFile, {{ ModelList::DownloadErrorRole, error }});
```